apps/condo/domains/meter/tasks/sendSubmitMeterReadingsPushNotifications.js

```diff
@@ -49,7 +49,7 @@ const readMeterReadings = async ({ context, meters }) => {
     // then calculate current period window
     const now = dayjs()
     const startWindowDate = `${now.format('YYYY-MM')}-01`
-    const endWindowDate = now.toISOString()
+    const endWindowDate = now.set('hour', 23).set('minute', 59).set('second', 59).toISOString()
```

Comment:
There is a method  `endof('day')` in dayjs
https://day.js.org/docs/en/manipulate/end-of